src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -1,4 +1,18 @@
 import Raven from 'raven-js';
+import hcaManifest from '../../hca/manifest.js';
+import dependentStatusManifest from '../../disability-benefits/686/manifest.js';
+import feedbackManifest from '../../edu-benefits/feedback-tool/manifest.js';
+import burialsManifest from '../../burials/manifest.js';
+import edu1990Manifest from '../../edu-benefits/1990/manifest.json';
+import edu1995Manifest from '../../edu-benefits/1995/manifest.json';
+import edu1990eManifest from '../../edu-benefits/1990e/manifest.json';
+import edu1990nManifest from '../../edu-benefits/1990n/manifest.json';
+import edu5490Manifest from '../../edu-benefits/5490/manifest.json';
+import edu5495Manifest from '../../edu-benefits/5495/manifest.json';
+import edu0993Manifest from '../../edu-benefits/0993/manifest.json';
+import preneedManifest from '../../pre-need/manifest.json';
+import pensionManifest from '../../pensions/manifest.json';
+import disability526Manifest from '../../disability-benefits/526EZ/manifest.json';
```

Comment:
Should we also plan to move the form benefits and form titles into the manifests as well, as part of a separate PR?

Reply:
@cehsu Yeah, I think that's a good idea.